applications/vanilla/models/class.discussionmodel.php

```diff
@@ -716,11 +685,6 @@ public function calculate(&$Discussion) {
 
         $Discussion->CountCommentWatch = is_numeric($Discussion->CountCommentWatch) ? $Discussion->CountCommentWatch : null;
 
-        if ($Discussion->LastUserID == null) {
-            $Discussion->LastUserID = $Discussion->InsertUserID;
-            $Discussion->LastDate = $Discussion->DateInserted;
-        }
-
```

Comment:
You can see here that we set a user in the case of a discussion with no comments. Removing this kind of logic could lead to theme breakage.